modules/auxiliary/gather/external_ip.rb

```diff
@@ -29,12 +29,19 @@ def initialize
     register_options(
       [
         Opt::RHOST('ifconfig.me'),
+        OptString.new('VHOST', [false, "HTTP server hostname", 'ifconfig.me']),
```

Comment:
Instead of defining it as a new option, you should set it as the default using:
```
'DefaultOptions' => { 'VHOST' => 'ifconfig.me' },
```

Put that under the References key in the module metadata and it should do the trick.